ledger/complete/wal/checkpointer_test.go

```diff
@@ -57,14 +57,14 @@ var (
 	logger             = zerolog.Logger{}
 	segmentSize        = 32 * 1024
 	pathByteSize       = 32
-	pathFinderVersion  = uint8(0)
+	pathFinderVersion  = uint8(1)
```

Comment:
why is this updated?

Reply:
shouldn't happen in this PR, I'll revert it back to 0.

Reply:
Fixed at c3cc915